pxtblocks/fields/field_note.ts

```diff
@@ -185,7 +235,10 @@ namespace pxtblockly {
          * @param {string} note The new note in string format.
          */
         setValue(note: string) {
-            note = String(parseFloat(note || "0"));
+            let regex: RegExp = new RegExp("Note.(.*)");
+            let match = regex.exec(note);
+            let noteName: any = (match && match.length > 1) ? match[1] : null;
```

Comment:
We typically avoid using the `any` type anywhere in the code. I think you can just declare `string` here.